.ios/Podfile

```diff
@@ -31,7 +32,6 @@ target 'Runner' do
   flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
 
   pod 'FlutterPluginRegistrant', :path => File.join('Flutter', 'FlutterPluginRegistrant'), :inhibit_warnings => true
-  pod 'FirebaseFirestore', :git => 'https://github.com/invertase/firestore-ios-sdk-frameworks.git', :tag => '10.9.0'
```

Comment:
can you put this back and change it to 10.10.0. We need this otherwise the build will run for 30 min

Reply:
I changed it @vusters